index.js

```diff
@@ -209,6 +213,60 @@ app.get("/popup", function (req, res) {
   res.status(200).send(popupMsg);
 });
 
+const modifyCoordinates = originalResponse =>
+  originalResponse.reduce((acc, curr) => {
+    const {
+      coordinates: {
+        coordinates: [longitude, latitude]
+      }
+    } = curr;
+
+    return [
+      ...acc,
+      {
+        ...curr,
+        coordinates: {
+          latitude,
+          longitude
+        }
+      }
+    ];
+  }, []);
```

Comment:
This could be just part of the new middleware function and using `.map(...)` is better as it creates a single new array instead of creating a new array for every element in the source array including all previous elements already reduced because of `[...acc, ...]`.